crates/bevy_ecs/src/system/exclusive_system.rs

```diff
@@ -9,8 +9,38 @@ pub trait ExclusiveSystem: Send + Sync + 'static {
 
     fn id(&self) -> SystemId;
 
+    /// Runs the exclusive system in the world.
+    ///
+    /// Use [`run_direct`] instead if you are manually running a system outside of a schedule
+
     fn run(&mut self, world: &mut World);
 
+    /// Runs the exclusive system directly on the world, initializing the world correctly
+    ///
```

Comment:
For both methods, it may be useful to describe how to persistently store and then access a system, rather than initializing it every frame.

This reduces work by enabling query caching and also allows `Local` resources to persist correctly.

I'm just not sure that the API docs are the right place to put that; it feels like content that might be better suited to the book.

Reply:
Actually, I suppose that the way to do this is to probably store the system, run `.initialize` once, and then call the old `.run()` manually.

The new method is much more convenient, but will probably perform worse for repeated invocations.

Hmm. Maybe we also demonstrate how to store and run a stage in the same section of the book?